js/tools.js

```diff
@@ -706,6 +706,18 @@ function getStorageAvailable() {
 	}
 }
 
+/**
+ * Refresh bulk actions combination number after creating or deleting combinations
+ *
+ * @param sign - int
```

Comment:
Please see [here](http://usejsdoc.org/tags-param.html) for the correct syntax for `@param`

Remember that `int` is not a valid javascript type (but `number` is)